src/components/nav/NavLink.js

```diff
@@ -1,9 +1,11 @@
-import React, {useEffect, useState} from 'react';
+import React, {useEffect, useState, useMemo} from 'react';
```

Comment:
`useMemo` not used in this file afaict

Reply:
Right - I didn't do `npm run format` yet.  Still lots of files to change :-)